Controller/CRUDController.php

```diff
@@ -265,6 +265,9 @@ public function editAction($id = null)
 
             // persist if the form was valid and if in preview mode the preview was approved
             if ($isFormValid && (!$this->isInPreviewMode() || $this->isPreviewApproved())) {
+                $object = $form->getData();
+                $this->admin->setSubject($object);
```

Comment:
Hmm... this makes no sense, unless `$form` does not return the same object. Maybe it clones it? Can you find out whether the object line 268 and the object line 256 have the same `spl_object_hash` or not?

I means [as you can see, there should be no need to get the object again if you want the modifications done by the form](https://3v4l.org/Gj6SB)